pkg/openstack/loadbalancer.go

```diff
@@ -515,10 +516,34 @@ func (lbaas *LbaasV2) createOctaviaLoadBalancer(service *corev1.Service, name, c
 		createOpts.VipAddress = loadBalancerIP
 	}
 
+	for _, port := range service.Spec.Ports {
+		listenerCreateOpt := lbaas.buildListenerCreateOpt(port, svcConf)
+		members, newMembers, err := lbaas.buildBatchUpdateMemberOpts(port, nodes, svcConf)
+		if err != nil {
+			return nil, err
+		}
+		poolCreateOpt := lbaas.buildPoolCreateOpt(string(listenerCreateOpt.Protocol), service, svcConf)
+		poolCreateOpt.Members = members
+		var withHealthMonitor string
+		if svcConf.enableMonitor {
+			opts := lbaas.buildMonitorCreateOpts(port)
+			poolCreateOpt.Monitor = &opts
+			withHealthMonitor = " with healthmonitor"
+		}
+
+		listenerCreateOpt.DefaultPool = &poolCreateOpt
+		createOpts.Listeners = append(createOpts.Listeners, listenerCreateOpt)
+		klog.V(2).Infof("Loadbalancer %s: adding pool%s using protocol %s with %d members", name, withHealthMonitor, poolCreateOpt.Protocol, len(newMembers))
```

Comment:
s/pool%s/pool %s ?

Reply:
This is fine, `withHealthMonitor` (`pool%s` vararg) is just an optional string that squeezed into the log message, `pool %s` would cause an double whitespace in case of `withHealthMonitor = ""`